google/cloud/forseti/scanner/audit/blacklist_rules_engine.py

```diff
@@ -220,6 +220,12 @@ def find_violations(self, instance_network_interface):
             project = network_and_project.group(1)
             network = network_and_project.group(2)
 
+            if not network_interface.access_configs:
+                LOGGER.warn('Network interface: %s, doesn\'t '
```

Comment:
I would like this message to show that we are unable to find the violation here.  So, something like:
```
Unable to determine blacklist violation for network interface: %s, because it doesn't have access_configs.
```

Reply:
Updated